module/data/actors/actor-kite.mjs

```diff
@@ -0,0 +1,99 @@
+import { CORIOLIS_TGD } from "../../config/config.mjs";
+import { DataHelper } from "../../helpers/data.mjs";
+import RollDialogWithConnectedActorAutomation from "../automations/roll-dialog-with-connected-actor.mjs";
+import RollDialogAutomation from "../automations/roll-dialog.mjs";
+import cgdVehicle from "./actor-vehicle.mjs";
+
+export default class cgdKite extends cgdVehicle {
+  static LOCALIZATION_PREFIXES = [
+    ...super.LOCALIZATION_PREFIXES,
+    'CORIOLIS_TGD.Actor.Kite',
+    'CORIOLIS_TGD.Item.Equipment',
+  ];
+
+  static defineSchema() {
+    const fields = foundry.data.fields;
+    const schema = super.defineSchema();
+
+    schema.bonus = new fields.NumberField({ ...DataHelper.requiredInteger, initial: 0, min: 0 });
```

Comment:
@robkuijper I am checking here and I think that the "bonus" that they have on the table is similar to Maneuverability that all the other vehicles have and that comes directly from the `cdgVehicle`.

I believe that we should remove this field and change the other points to use the Maneuverability instead.

Example on hover:

<img width="524" height="92" alt="Image" src="https://github.com/user-attachments/assets/b2a87710-6a7a-4683-9ed4-cc39fddc5836" />

Although they do not call it "bonus", they have the same logic of `+X` value.

Reply:
I've made the required changes; I'm keeping the `Use Kite` automation, since (unlike the Stunt automation) it allows you to pick an attribute/stat to roll with. Does this sound good? 🙂